core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -615,9 +615,9 @@ package object config {
     .stringConf
     .createOptional
 
-  private[spark] val METRICS_REGISTER_STATIC_SOURCES_ENABLED =
-    ConfigBuilder("spark.metrics.register.static.sources.enabled")
-      .doc("Whether to register static sources with the metrics system")
+  private[spark] val METRICS_STATIC_SOURCES_ENABLED =
+    ConfigBuilder("spark.register.static.sources.enabled")
```

Comment:
Hi, @LucaCanali . This must be `spark.metrics.static.sources.enabled`~ :)

Reply:
Could you make a small UT for this?
I guess this typo is not detected because we don't have a UT.

Reply:
Indeed, thanks. I will add the UT, I have just pushed the corrected config parameter name.